src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/OOPTagHelperResolver.cs

```diff
@@ -4,8 +4,10 @@
 #nullable disable
```

Comment:
The `OOPTagHelperResolver` is the component that sits in `devenv.exe` and talks to the OOP component. In this PR its responsibility is to ask the ServiceHub process for TagHelper deltas and then transform those delta back into normal TagHelperResolution results that the rest of the system can use. Basically it caches the last received TagHelper result and applies deltas to it when responding to future requests.